### PR TITLE
New Look and Feel for Proof of Humanity

### PR DESCRIPTION
**Pull request by santisiri**
The new UI has an improved look and feel for the Open & Democratic Proof of Humanity. Changes include:
- New logo and icon that works well at all resolutions.
- Dark mode by default user experience
- Improved navigation of buttons & upload wizard
- Links to all of the resources created exclusively by the DAO 
- Ready to work with Open & Democratic PoH v2 fork.

### CODE REVIEW

File: components/accordion.js [hunk 1]
```diff
@@ -55,23 +57,26 @@ function AnimatedAccordionItemHeading({ expanded, children, ...rest }) {
...
   return (
     <Box as={_AccordionItemHeading} {...rest}>
       <Box
         as={_AccordionItemButton}
         variant="accordion.heading"
         sx={{
-          backgroundImage({ colors: { accent, accentComplement } }) {
-            return `linear-gradient(90deg, ${accent} 0%, ${accentComplement} 100%, ${accentComplement} 100%)`;
-          },
+          backgroundImage: `linear-gradient(90.13deg, var(--theme-ui-colors-accent, ${accColor}) 49.89%, var(--theme-ui-colors-accentComplement, #517aea) 93.37%)`,
```
Comment (bilinkis) [inline components/accordion.js:67]: Not sure I like how these colors look like. I'd probably stick to the orange color identity.

**Review by bilinkis — COMMENTED** [review]
Never used Ethermail.io, but it looks to be beta and not open yet for everyone.

File: components/accordion.js (hunk 1 above)
Comment (bilinkis) [inline components/accordion.js:67]: Not sure I like how these colors look like. I'd probably stick to the orange color identity.

File: components/account-settings-popup.js [hunk 2]
```diff
@@ -90,10 +89,14 @@ export default function AccountSettingsPopup({
...
   return (
     <Popup
-      contentStyle={{ width: 490, lineHeight: "initial" }}
+      contentStyle={{ width: 490, lineHeight: "initial", cursor: "pointer" }}
       trigger={
-        <IconButton>
-          <Settings />
+        <IconButton sx={{ cursor: "pointer", marginTop: "3px", width: "40px" }}>
+          <Image
+            src="/images/ethereum.svg"
```
Comment (bilinkis) [inline components/account-settings-popup.js:96]: Not sure this icon reflects what the menu is for, but the menu is pretty useless altogether, so might as well delete it.

File: components/header.js [hunk 3]
```diff
@@ -224,30 +224,22 @@ function DesktopNavbar() {
...
-            </Box>
-          */}
       </Col>
       <Col className="poh-header-menu" span={17}>
         <Row justify="center">
-          <Col span={6} className="poh-header-item">
+          <Col span={15} className="poh-header-item">
             <NextLink href="/" as="/">
               <Link className="poh-header-text" variant="navigation">
                 {t("header_profiles")}
```
Comment (bilinkis) [inline components/header.js:240]: Design looks good but not centered within the navigation bar

File: _pages/profile/submit/submit-profile/finalize-tab.js [hunk 4]
```diff
@@ -83,7 +84,7 @@ class _FinalizeTab extends React.Component {
                   <Paragraph>
                     People can try to notify you of problems in your submission
                     and save your deposit via your{" "}
-                    <Link href="https://ethmail.cc/">ethmail.cc</Link>. Make sure
+                    <Link href="https://ethermail.io/">ethermail.io</Link>. Make sure
                     to check it while submission is being processed.
                   </Paragraph>
                 </>
```
Comment (bilinkis) [inline _pages/profile/submit/submit-profile/finalize-tab.js]: Color of the name element should be white instead of black

**Review by bilinkis — CHANGES_REQUESTED** [review]

File: components/accordion.js (hunk 1 above)
Comment (santisiri) [reply components/accordion.js:67]: Those colors are on the same gamma that we are using across the board and brings in a 2nd set of colors that can be used to contextualize better the information specifically about challenges. I will keep on working on the "challenges" menu, specially as we connect to a Court provider on Polygon and the networks we'll deploy for the v2.

File: components/account-settings-popup.js (hunk 2 above)
Comment (santisiri) [reply components/account-settings-popup.js:96]: It's really a menu about handling your Ethereum connection to the dapp (not really about Settings). That's why we use the ETH icon. That said: I will keep polishing that menu there on subsequent versions.

File: components/header.js (hunk 3 above)
Comment (santisiri) [reply components/header.js:240]: It's attached to the grid system that the application uses overall, and we must also make sure the relevant buttons appear on mobile screens.

File: _pages/profile/submit/submit-profile/finalize-tab.js (hunk 4 above)
Comment (santisiri) [reply _pages/profile/submit/submit-profile/finalize-tab.js]: Will fix on a subsequent pull.

File: _pages/profile/[id]/submission-details-accordion.js [hunk 5]
```diff
@@ -125,7 +125,7 @@ export default function SubmissionDetailsAccordion({ submission, contract }) {
       )}
       <SubmissionDetailsAccordionItem
         heading={t("profile_details_evidence")}
-        panelSx={{ paddingX: 0 }}
+        panelSx={{ paddingX: 0, borderRadius: 12 }}
         panel={
           <Evidence
```
Comment (mmiguerodriguez) [inline _pages/profile/[id]/submission-details-accordion.js:130]: Nice to have: Add margin between evidence text and corresponding card.

File: _pages/profile/submit/submit-profile/finalize-tab.js [hunk 6]
```diff
@@ -75,6 +75,7 @@ class _FinalizeTab extends React.Component {
           <Col span={24}>
```
Comment (mmiguerodriguez) [inline _pages/profile/submit/submit-profile/finalize-tab.js]: `About me` on the finalize-tab is black and feels weird. Text should be white

**Review by mmiguerodriguez — CHANGES_REQUESTED** [review]
LGTM. Good work.
I'm still not completely sure about the green/blue colors on the submit-profile/evidence tab. 
Only changes needed @ `finalize-tab -> about me` .

**Comment by santisiri** [comment]
On the colors, here's a test of the palette we are using. Main color remains as the yellow trend since that has been the predominant color of PoH.. but we need secondary and tertiary colors to convey these are other smart contracts / environments we are interacting with in the long run. 

TL;DR: there's a plan. 
Screen Shot 2022-09-24 at 7 42 36 PM

File: _pages/profile/submit/submit-profile/finalize-tab.js (hunk 6 above)
Comment (santisiri) [reply _pages/profile/submit/submit-profile/finalize-tab.js]: Now I realize what you and @bilinkis meant. Here's a quick fix: https://github.com/OpenProofOfHumanity/proof-of-humanity-web/pull/6/commits/8579239a781d3b9df5ec8756830c3b22de1d0d66

And here's how it looks now: 
Screen Shot 2022-11-21 at 5 14 14 PM

File: _pages/profile/[id]/submission-details-accordion.js (hunk 5 above)
Comment (santisiri) [reply _pages/profile/[id]/submission-details-accordion.js:130]: Fixed with the latest push: https://github.com/bilinkis/proof-of-humanity-web/pull/3/commits/0a332745ed16a93008aa554daaa8ce5ba0b5480c 

Hope I understood you correctly, looks better imho now:
Screen Shot 2022-11-21 at 5 26 23 PM

File: _pages/profile/submit/submit-profile/finalize-tab.js (hunk 4 above)
Comment (santisiri) [reply _pages/profile/submit/submit-profile/finalize-tab.js]: Update: I will remove this and have a proper setup once Etheremail effectively deploys their PoH integration.

**Comment by santisiri** [comment]
@bilinkis @mmiguerodriguez I think I finished addressing your concerns. Let me know if there's anything else I should have a look. Thanks for your help!

**Review by mmiguerodriguez — APPROVED** [review]
LGTM! Great step improving the current UI/UX.
Hopefully in the future we can have a revamp/fixing in which we can address package versioning/compatibility issues we may not be tackling right now.

**Review by bilinkis — APPROVED** [review]
LGTM.
